plugins/inputs/teamspeak/README.md

```diff
@@ -0,0 +1,43 @@
+# Teamspeak 3 Input Plugin
+
+This plugin uses the Teamspeak 3 ServerQuery interface of the Teamspeak server to collect statistics of one or more
+virtual servers. If you are querying an external Teamspeak server, make sure to add the host which is running Telegraf
+to query_ip_whitelist.txt in the Teamspeak Server directory.
+
+### Configuration:
+
+```
+[[inputs.teamspeak]]
+## Server address for Teamspeak 3 ServerQuery
```

Comment:
Fix up the indention on the comments, make sure it matches the SampleConfig, more info about that further down.